<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>FOURNISSEUR</string>
       </attribute>
       <widget class="QTabWidget" name="tab_four">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>511</width>
          <height>501</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>ajouter un fournisseur</string>
         </attribute>
         <widget class="QLineEdit" name="nom_fournisseur_2">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>70</y>
            <width>151</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="prenom_fournisseur">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>140</y>
            <width>151</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="id">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>210</y>
            <width>151</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>440</y>
            <width>361</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>Ajouter</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>24</x>
            <y>70</y>
            <width>111</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Nom_dournisseur :</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>140</y>
            <width>131</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Prénom_fournisseur :</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>210</y>
            <width>121</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Id_fournisseur :</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_17">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>290</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Num_Tel :</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_18">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>340</y>
            <width>55</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>RIB:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="Num_fournisseur">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>280</y>
            <width>151</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="RIB_fournisseur">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>330</y>
            <width>151</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>modifier un fournisseur</string>
         </attribute>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>70</y>
            <width>121</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Nom_fournisseur :</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>140</y>
            <width>131</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Prénom_fournisseur :</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>230</y>
            <width>111</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Id_fournisseur :</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="nom_modifier">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>70</y>
            <width>171</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="prenom_modifier">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>140</y>
            <width>171</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="id_modifier">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>230</y>
            <width>171</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_3">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>440</y>
            <width>361</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>Modifier</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_20">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>310</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Num_Tel :</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_21">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>380</y>
            <width>55</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>RIB:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="Num_modifier">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>300</y>
            <width>151</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="RIB_modifier">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>380</y>
            <width>151</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>supprission d'un fournisseur</string>
         </attribute>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>130</y>
            <width>131</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>id_fournisseur</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>430</y>
            <width>361</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="id_sup">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>130</y>
            <width>161</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QTableView" name="table_four">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>30</y>
          <width>491</width>
          <height>331</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>940</x>
          <y>410</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>tri</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_r_f">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>490</y>
          <width>341</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>940</x>
          <y>490</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>recherche</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>PRODUIT</string>
       </attribute>
       <widget class="QTabWidget" name="table_pro">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>681</width>
          <height>561</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_6">
         <attribute name="title">
          <string>Ajouter_produit</string>
         </attribute>
         <widget class="QPushButton" name="pushButton_6">
          <property name="geometry">
           <rect>
            <x>410</x>
            <y>500</y>
            <width>251</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>Ajouter</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="nom_ajouter">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>80</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="quantiter_ajouter">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>180</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="id_ajouter">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>340</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>80</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Nom_produit :</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_13">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>180</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Qantité_produit :</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_14">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>270</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Date_Arivage :</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_15">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>340</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Id_produit :</string>
          </property>
         </widget>
         <widget class="QDateEdit" name="date_ajouter">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>270</y>
            <width>110</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_19">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>410</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Id_fournisseur :</string>
          </property>
         </widget>
         <widget class="QComboBox" name="list_fournisseur">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>390</y>
            <width>291</width>
            <height>41</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_8">
         <attribute name="title">
          <string>Modifier_produit</string>
         </attribute>
         <widget class="QPushButton" name="pushButton_5">
          <property name="geometry">
           <rect>
            <x>330</x>
            <y>490</y>
            <width>331</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>Modifier</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="nom_modifier_2">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>60</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="id_modifier_2">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>170</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>60</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Nom_produit :</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>170</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Id_produit :</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="quantiter_modifier">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>120</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>120</y>
            <width>111</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>quantiter_produit :</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_16">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>250</y>
            <width>111</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>date_produit : </string>
          </property>
         </widget>
         <widget class="QDateEdit" name="date_modifier_2">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>250</y>
            <width>110</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QComboBox" name="list_fournisseur_2">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>330</y>
            <width>291</width>
            <height>41</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_22">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>340</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Id_fournisseur :</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_7">
         <attribute name="title">
          <string>Supprimer_produit</string>
         </attribute>
         <widget class="QPushButton" name="pushButton_4">
          <property name="geometry">
           <rect>
            <x>330</x>
            <y>490</y>
            <width>331</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="id_4">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>190</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>190</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Id_produit :</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QTableView" name="tab_pro">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>20</y>
          <width>321</width>
          <height>251</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1068</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
